### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by leggsimon**

### CODE REVIEW

**Comment by gitGremlin** [comment]
💃 👍 👍 
giphy-1

**Comment by gitGremlin** [comment]
giphy-2

File: index.js [hunk 1]
```diff
@@ -14,7 +14,13 @@ const coolOff = mon => {
 	}
 }
 
-const getFlavourText = pokedexEntry => pokedexEntry.flavor_text_entries.find(entry => entry.language.name === 'en' && entry.version.name === 'alpha-sapphire').flavor_text;
+const getFlavourText = pokedexEntry => {
+	if (getName(pokedexEntry) === 'Pidgey') {
```
Comment (gitGremlin) [inline index.js]: `if (getName(pokedexEntry) === 'Pidgey'||'Rattata')`

Comment (leggsimon) [inline index.js]: 👍

File: index.js [hunk 2]
```diff
@@ -14,7 +14,13 @@ const coolOff = mon => {
 	}
 }
 
-const getFlavourText = pokedexEntry => pokedexEntry.flavor_text_entries.find(entry => entry.language.name === 'en' && entry.version.name === 'alpha-sapphire').flavor_text;
+const getFlavourText = pokedexEntry => {
+	if (getName(pokedexEntry) === 'Pidgey'||'Rattata') {
```
Comment (keithamus) [inline index.js]: Needs to be `if (getName(pokedexEntry) === 'Pidgey'|| getName(pokedexEntry) === 'Rattata')` or `if (getName(pokedexEntry).match(/Pidgey|Rattata/))`

Comment (leggsimon) [inline index.js]: Ahhhhh yess!

File: index.js [hunk 3]
```diff
@@ -14,7 +14,13 @@ const coolOff = mon => {
 	}
 }
 
-const getFlavourText = pokedexEntry => pokedexEntry.flavor_text_entries.find(entry => entry.language.name === 'en' && entry.version.name === 'alpha-sapphire').flavor_text;
+const getFlavourText = pokedexEntry => {
+	if (getName(pokedexEntry).match(/Pidgey|Rattata/)) {
```
Comment (gitGremlin) [inline index.js:18]: Zubat, Spearow, Drowzee

Comment (leggsimon) [inline index.js:18]: You shouldn't even be catching them with their 50 candy ridiculousness!

**Comment by apaleslimghost** [comment]
No.

**Pull request closed without merging**